kvmagent/kvmagent/test/test_snapshots.py

```
@@ -46,3 +48,117 @@ def test_snapshot_operations(self):
+            "volume": {
+                "installPath": memory_snapshot_path
+            },
+            "memory": True,
+            "live": True,
+            "full": False,
+        }
+
+        try:
+            vm_utils.take_volumes_snapshots(vm_uuid, [boot_disk_struct, memory_disk_struct])
+        except Exception as e:
+            pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)